usmqe_tests/api/gluster/conftest.py

```diff
@@ -41,25 +41,37 @@ def invalid_volume_id(request):
 
 
 @pytest.fixture
-def valid_volume_bricks():
+def valid_volume_configuration(valid_volume_name):
     """
-    Generate valid brick for api in format:
-        hostname:brick_path
+    Generate valid configuration for volume creation with set:
+        "Volume.volname", "Volume.bricks", "Volume.replica_count", "Volume.force"
     """
     role = pytest.config.getini("usm_gluster_role")
     try:
-        return ["{}:{}".format(x, pytest.config.getini(
-            "usm_brick_path")) for x in inventory.role2hosts(role)]
+        bricks = [[{"{}".format(inventory.role2hosts(role)[i]):
+                    "{}".format(pytest.config.getini("usm_brick_path"))},
+                   {"{}".format(inventory.role2hosts(role)[i+1]):
+                    "{}".format(pytest.config.getini("usm_brick_path"))}]
+                  for i in range(0, len(inventory.role2hosts(role)), 2)]
     except TypeError as e:
         print(
             "TypeError({0}): You should probably define usm_brick_path and \
                     usm_gluster_role in usm.ini. {1}".format(
                 e.errno,
                 e.strerror))
+    return {
+        "Volume.volname": valid_volume_name,
+        "Volume.bricks": bricks,
+        "Volume.replica_count": "2",
+        "Volume.force": True}
 
 
-@pytest.fixture(params=[None, "0000000000000000"])
-def invalid_volume_bricks(request):
+@pytest.fixture(params=[{
+    "Volume.volname": "Volume_invalid",
+    "Volume.bricks": None,
```

Comment:
When some attribute is set to None then in request json is set to `null`.
e.g. `{"Volume.replica_count": "2", "Volume.bricks": null, "Volume.volname": "Volume_invalid", "Volume.force": true}`